etc/Dockerfile.base

```diff
@@ -18,7 +18,6 @@ RUN apt-get -y --no-install-recommends install \
     gnupg \
     gpg \
     less \
-    libabsl-dev \
```

Comment:
no longer needed for sdk? does conan take care of it?

Reply:
Not needed, matching up dep list in the viam-cpp-sdk[ jammy container](https://github.com/viamrobotics/viam-cpp-sdk/blob/main/etc/docker/base-images/Dockerfile.ubuntu.jammy).